docs/cloud/aws/lca1/customer-ssp/index.md

```diff
@@ -0,0 +1,97 @@
+An SSP is a document that outlines how an organization protects the security of a system.
+An SSP is a critical document that helps organizations meet compliance requirements.
+
+The [OIT IT Security](https://oit.colorado.edu/services/it-security) group is responsible for guiding and evaluating your SSP.
+[OIT IT Security](https://oit.colorado.edu/services/it-security) prescribes the [NIST SP 800-171 Rev. 2](https://csrc.nist.gov/pubs/sp/800/171/r2/upd1/final)
+control set as the official standard to meet compliance requirements for **Confidential Information** and **Highly Confidential Information**.
+
+Getting your SSP completed and approved is not only necessary, but can be a rigorous process.
+The good news is that by using the OIT Research Computing [Cloud Foundations Service](https://www.colorado.edu/rc/cloudfoundations) (CFS)
+**LCA1** solution, some of your security burden is handled by our service.
+
+The CFS **LCA1** solution is an [AWS Landing Zone](https://docs.aws.amazon.com/prescriptive-guidance/latest/migration-aws-environment/understanding-landing-zones.html)
+that is approved to support **Confidential Information**.  Support for **Highly Confidential Information**
+is on the horizon.
+
+The rest of this document will discuss how the CFS service can help you meet some of your security requirements.
+We also have a quickstart guide to help you get started on your SSP journey.
+
+## Contents
+* [Shared Responsibility Model](#shared-responsibility-model)
+* [CFS **LCA1** Features](#cfs-lca1-features)
+* [SSP Quickstart Process](#ssp-quickstart-process)
+
+### Shared Responsibility Model
+Start with understanding the [Shared Responsibility Model](../getting-started/shared-responsibility-model.md).
+It is important to know that the cloud provider, our service and you as the cloud customer all share the burden of securing systems.
+
+### CFS **LCA1** Features
+The OIT Research Computing [Cloud Foundations Service](https://www.colorado.edu/rc/cloudfoundations)
+(CFS) team owns and manages an [AWS Landing Zone](https://docs.aws.amazon.com/prescriptive-guidance/latest/migration-aws-environment/understanding-landing-zones.html)
+specifically designed to enable support of the [NIST SP 800-171 Rev. 2](https://csrc.nist.gov/pubs/sp/800/171/r2/upd1/final)
+control set.  By using the CFS **LCA1** service, we help you meet some of your security controls.
+Below is a list of features we offer that can help you with your SSP.
+
+```{important}
+Using the CFS **LCA1** service does not automatically make your system compliant.  Our system is a foundation that enables you to be compliant.
+```
+
+**Federated Identity and Access Management (IAM)**
+
+When using the CFS **LCA1** landing zone, role access is integrated with the CU Boulder
+[OIT Identity and Access Management](https://oit.colorado.edu/services/identity-access-management)
+(IAM) offering.  This offering gives you:
```

Comment:
Should we also mention group audit logging (via Grouper) here?

Reply:
I will add something like "The role we provide you is set up with temporary short-term credentials to help you meet compliance."